tests/trainer/test_trainer.py

```diff
@@ -1748,6 +1748,21 @@ def test_torchdynamo_full_eval(self):
         metrics = trainer.evaluate()
         self.assertAlmostEqual(metrics["eval_loss"], original_eval_loss)
 
+        # 4. TorchDynamo fx2trt
```

Comment:
1. The test passed because we set the rule in TorchDynamo as if the TRT lowering by torch_tensorrt.fx is failed, the model will fall back eager mode. So I think that is reason the test still works.
2. Good question
> which of course begs the question - where in the tests do you check if torch_tensorrt is available?

I will add a `try` block to check if `import torch_tensorrt.fx` is correctly returned.

Reply:
Yeah, let's figure out how to not have an assert and a passing test at the same time ;) Thank you!

Reply:
but also if the user failed to install tensor RT properly (like I did) and now the API hides that - how will the user know that their setup is incomplete/broken?

Reply:
Actually, similar to other library like pytorch, users will be alerted by
1) in our installation, we asked user to try `python -c "import torch_tensorrt.fx"` or they will be alerted by stdout by
ModuleNotFoundError: No module named 'torch_tensorrt'
2) if fx lowering is failed, a print statement is there like 
https://github.com/pytorch/torchdynamo/blob/5fb502660e52a2e1f93ab0f148fd8776e1b56297/torchdynamo/optimizations/backends.py#L742

Reply:
one question, is this `test_trainer` in CI/CD? Should we add torch_tensorrt.fx as one dependency of environment setup? 
Or even have `@require_torchtensorrt.fx` ?

Reply:
re: diagnostics - please let's save all your notes in the docs -> or if they are already in your docs then direct links please (i.e. "here is how to diagnose things")

Reply:
Will add the user self-diagnostics to docs.